### ISSUE TITLE
Question about the loss function

### ISSUE THREAD

**Issue by LeeBC2298** [vyom1911/Facial-Landmark-Detection#2]
I think the facial landmark regression loss and other losses should add together,
![image](https://user-images.githubusercontent.com/41717225/84107305-db80a180-aa4f-11ea-9cc8-96afe270bcbb.png)
In the paper there is a subtraction because softmax cross entropy losses are negative.

**Comment by jinneigeyu** [comment]
I agree. and the papper says "when  the  gender  , smile , glass , pose  branch model  are trainning good, these branches' parmas don't need to train ". These part  wan not shown in this implementation
